go/vt/mysqlctl/backupengine.go

```diff
@@ -193,51 +205,198 @@ type BackupManifest struct {
 	// Position is the replication position at which the backup was taken.
 	Position mysql.Position
 
+	// PurgedPosition stands for purged GTIDs, information that is necessary for PITR recovery. This is specific to MySQL56
+	PurgedPosition mysql.Position
+
+	// FromPosition is only applicable to incremental backups, and stands for the position from
+	// which incremental changes are backed up.
+	FromPosition mysql.Position
+
+	// Incremental indicates whether this is an incremental backup
+	Incremental bool
+
 	// BackupTime is when the backup was taken in UTC time (RFC 3339 format)
 	BackupTime string
 
 	// FinishedTime is the time (in RFC 3339 format, UTC) at which the backup finished, if known.
 	// Some backups may not set this field if they were created before the field was added.
 	FinishedTime string
+
+	// ServerUUID identifies the server from which backup was taken
+	ServerUUID string
+
+	TabletAlias string
+
+	Keyspace string
+
+	Shard string
```

Comment:
Are the changes to the manifest backwards compatible? I.e. can I take a full backup on v16, downgrade to v15 and restore from that backup?

Reply:
@deepthi yes, you can. The added fields are not actually read/used for now (with the exception of PITR-specific data). For now, they're added as means of visibility and simpler means for debugging and tracing "what happened where and by whom".